### DISCUSSION TITLE
Issue with OGCS automatically opening Outlook

### DISCUSSION THREAD

**Discussion by RPross** [phw198/OutlookGoogleCalendarSync#1940] [General]
[presently using v2.11.0.0]

Hello,

I’ve been encountering a recurring issue with Outlook Google Calendar Sync (OGCS) over the past few months, regardless of the version I use. Here’s the situation:

When I start my Windows 10 computer, OGCS starts automatically as expected. However, in the background, it also triggers Outlook to open, or sometimes just Outlook's Calendar. Once started, both OGCS and Outlook (or its Calendar) icons appear in the taskbar **tray**, but the Outlook icon on the actual taskbar itself is not highlighted and remains inaccessible until I manually click-start the program.

This issue is problematic because it can result in multiple instances of Outlook attempting to run in the background simultaneously. This leads to warning messages and requires me to manually close the other instances via Task Manager.

**My question:** How can I configure OGCS to prevent it from automatically opening Outlook? Ideally, I would like OGCS to sync without launching Outlook. If that’s not feasible, I’m willing to wait to sync calendars until I manually start Outlook, as this seems to be a straightforward workaround.

Everything else with OGCS is functioning correctly.

Thank you for your assistance.

**Comment by phw198** [comment]
I didn't think multiple instances of Outlook was possible - I'd be interested to see those warning messages when that occurs and also your OGCS log files for the time period too, to see exactly what's happening.

If Outlook is not running when OGCS starts to sync, it has to open Outlook, but you would see a small Oultlook icon with a cog on it in the task tray:
![image](https://github.com/user-attachments/assets/4150d05a-0f77-457d-a072-92d4eaa1a071)

Once OGCS has finished syncing, this icon should then disappear again, so the sync is relatively unobtrusive. 

When Outlook opens for you, is there any user input required? Eg you have to select an email profile or something, which is stopping OGCS from working properly?
